ui/server/src/test/scala/pl/touk/nussknacker/ui/api/ServiceRoutesSpec.scala

```diff
@@ -70,23 +69,24 @@ class ServiceRoutesSpec extends FunSuite with Matchers with ScalatestRouteTest w
       """.stripMargin)
     Post("/service/streaming/enricher", entity) ~> serviceRoutes.securedRoute ~> check {
       status shouldEqual StatusCodes.InternalServerError
-      entityAs[JsonThrowable].message shouldEqual Some("NonEmptyList(ExpressionParseError(EL1041E: After parsing a valid expression, there is still more data in the expression: 'spell',defaultNodeId,Some(param),not valid spell expression))")
-      entityAs[JsonThrowable].className shouldEqual classOf[ParametersCompilationException].getCanonicalName
+      entityAs[JsonThrowable].message shouldEqual Some("MissingParameters(Set(tariffType),)")
```

Comment:
What about adding tariffType parameter to the invocation so the error should remain more or less the same?

Reply:
:+1: done